### DISCUSSION TITLE
Support for views in Text tag?

### DISCUSSION THREAD

**Discussion by SupertigerDev** [microsoft/react-native-windows#12944] [Q&A] [answered]
Hello, when is support for views in Text tag being added? I want to port my chat app to react native for windows but I need this to support markdown. For example when tagging users, I need to show a small profile picture of the user inline. Is it possible to do it another way?

**Comment by chrisglein** [comment, marked as answer]
These two issues track this:
- https://github.com/microsoft/react-native-windows/issues/6315
- https://github.com/microsoft/react-native-windows/issues/5689

You can nest some things today - there's an example of nested Text components in [React Native Gallery](https://github.com/microsoft/react-native-gallery/)), for example. So... may depend on what you've tried?

But I think the short answer is that on the current "Paper" rendering architecture, this will remain unimplemented except for the cases that work today. But this should be solved for the newer "Fabric" renderer that will replace it.

**Reply by SupertigerDev** [reply]
![image](https://github.com/microsoft/react-native-windows/assets/8193841/46b6b7a1-2317-4b0a-8fbf-a476e6f9696a)
I'm assuming youre talking about this page of the app. This isnt really inline, this is just 2 views side by side in a flexbox. it will wont work well for a chat app as it will break new lines

**Reply by SupertigerDev** [reply]
Example: https://snack.expo.dev/UbgNDVV0SElLj5VrMl2ES

**Reply by chrisglein** [reply]
I was more referring to this example:
![image](https://github.com/microsoft/react-native-windows/assets/26607885/b841fe53-f2ac-465e-a45c-186bebb0cebe)

Which is admittedly `Text` within `Text`, not a custom component.

**Reply by SupertigerDev** [reply]
ah i see, and yes 😭

**Reply by SupertigerDev** [reply]
![image](https://github.com/microsoft/react-native-windows/assets/8193841/ea9759b4-164d-41ff-a1cf-116254bde363)
seems like even on fabric, i still get this message :(

**Reply by chrisglein** [reply]
I've opened https://github.com/microsoft/react-native-windows/issues/12997 to investigate.
